src/controllers/admin.controller.js

```diff
@@ -44,65 +48,42 @@ adminCtrl.addUser = async(req, res) => {
             user_password,
             user_status,
         } = req.body;
-        const userErrors = [];
         
-        // Validations
-        if (!user_name) {
```

Comment:
Why did u remove those lines?

Reply:
we have removed in order to take the validation from the service, we made testing and the validations still happens.